packages/back-end/src/api/organizations/postOrganization.ts

```diff
@@ -0,0 +1,47 @@
+import { freeEmailDomains } from "free-email-domains-typescript";
+import { PostOrganizationResponse } from "../../../types/openapi";
+import {
+  createApiRequestHandler,
+  validateIsSuperUserRequest,
+} from "../../util/handler";
+import { postOrganizationValidator } from "../../validators/openapi";
+import {
+  createOrganization,
+  toOrganizationApiInterface,
+} from "../../models/OrganizationModel";
+
+export const postOrganization = createApiRequestHandler(
+  postOrganizationValidator
+)(
+  async (req): Promise<PostOrganizationResponse> => {
+    const user = await validateIsSuperUserRequest(req);
+
+    const { name } = req.body;
+
+    if (name.length < 3) {
+      throw Error("Name length must be at least 3 characters");
+    }
+
+    let verifiedDomain = "";
+
+    // if the owner is verified, try to infer a verified domain
+    if (user.email && user.verified) {
+      const domain = user.email.toLowerCase().split("@")[1] || "";
+      const isFreeDomain = freeEmailDomains.includes(domain);
+      if (!isFreeDomain) {
+        verifiedDomain = domain;
+      }
+    }
```

Comment:
I don't think we want to set verified domains as part of this REST endpoint.  That's more for a user registering an organization for themselves